<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OreFileNameForm</class>
 <widget class="QWidget" name="OreFileNameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File name format</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <widget class="QLineEdit" name="fileNameFormatLineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="formatSpecifierComboBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>%%  The % character</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%o  Recording number (integer, always increasing)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%O  Recording number, 6 digits (000001, 000002...)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%a  Weekday, abbreviated (Mon, Tue, Wed...)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%A  Weekday, full (Monday, Tuesday, Wednesday...)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%d  Day of the month, 2 digits (01-31)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%e  Day of the month (1-31)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%m  Month number, 2 digits (01-12)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%b  Month name, abbreviated (Jan, Feb, Mar...)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%B  Month name, full (January, February, March...)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%y  Year, 2 digits (00-99)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%Y  Year, 4 digits (1900-2100)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%l  Hour, 12h (1-12)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%I  Hour, 12h, 2 digits (01-12)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%k  Hour, 24h (0-23)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%H  Hour, 24h, 2 digits (00-23)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%p  AM or PM, uppercase (note reversed case of p and AM/PM)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%P  am or pm, lovercase (note reversed case of P and am/pm)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%M  Minute, 2 digits (00-59)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%s  Seconds since 00:00:00 1970-01-01 UTC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%S  Second, 2 digits (00-59)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="promptLabel">
     <property name="text">
      <string>Resolved format string:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="validatedFileNameLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <item>
      <widget class="QPushButton" name="pushButtonValidate">
       <property name="text">
        <string>Validate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
